src/libraries/System.IO.Compression/src/System/IO/Compression/DeflateZLib/Inflater.cs

```diff
@@ -17,6 +17,7 @@ internal sealed class Inflater : IDisposable
         private const int MinWindowBits = -15;              // WindowBits must be between -8..-15 to ignore the header, 8..15 for
         private const int MaxWindowBits = 47;               // zlib headers, 24..31 for GZip headers, or 40..47 for either Zlib or GZip
 
+        private bool _nonZeroInput;                         // Whether there is any non zero input
```

Comment:
nit, would nonEmptyInput be better name? a zero input might be a buffer of zeroes.
or perhaps this means nonEmptyOutput?

Reply:
Changed to nonEmptyInput.